tutorials/notebooks/example_pytorch_ssdlite_mobilenetv3.ipynb

```diff
@@ -0,0 +1,459 @@
+    "            else:\n",
+    "                head_outputs = {\"bbox_regression\": head_outputs[1],\n",
+    "                                \"cls_logits\": head_outputs[0]}\n",
+    "\n",
+    "        detections = model.postprocess_detections(head_outputs, anchors, image_list.image_sizes)\n",
+    "        detections = model.transform.postprocess(detections, image_list.image_sizes, original_image_sizes)\n",
+    "        return detections\n",
+    "\n",
+    "\n",
+    "postprocess = PostProcess()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25ef70dd-d513-48ae-b7d0-4e1cac164d06",
+   "metadata": {},
+   "source": [
+    "## Dataset preparation\n",
+    "\n",
+    "Assuming we've downloaded the COCO dataset to a folder, let's set the folder path:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8bd5d8af-7cbd-4b1c-9b20-aac316b7bbe9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "TRAIN_DATASET_FOLDER = '/path/to/coco/training/images/train2017'\n",
+    "TRAIN_DATASET_ANNOTATION_FILE = '/path/to/coco/annotations/instances_train2017.json'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf80ebaf-7ae6-4b34-ae48-8463fc47a40d",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Now, let's create two functions. The first is for preprocessing the dataset and the second is for creating an unlabeled representative dataset for quantization calibration. We will use a batch size of 16 for quantization:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e92968e1-cd96-44a1-9ced-bcefad721de2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def train_collate_fn(batch_input):\n",
+    "    # collating images for the quantized model should return a single tensor: [B, C, H, W]\n",
+    "    images = torch.stack([b[0] for b in batch_input])\n",
+    "    targets = [b[1] for b in batch_input]\n",
+    "    return images, targets\n",
+    "\n",
+    "\n",
+    "coco_train = torchvision.datasets.CocoDetection(root=TRAIN_DATASET_FOLDER, annFile=TRAIN_DATASET_ANNOTATION_FILE,\n",
+    "                                                transforms=preprocess)\n",
+    "train_loader = torch.utils.data.DataLoader(coco_train, batch_size=16, shuffle=False, num_workers=0,\n",
+    "                                           collate_fn=train_collate_fn)\n",
+    "\n",
+    "coco_eval = torchvision.datasets.CocoDetection(root=EVAL_DATASET_FOLDER, annFile=EVAL_DATASET_ANNOTATION_FILE,\n",
+    "                                               transforms=preprocess)\n",
+    "eval_loader = torch.utils.data.DataLoader(coco_eval, batch_size=50, shuffle=False, num_workers=0,\n",
+    "                                          collate_fn=train_collate_fn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1d769fc-0c8f-40ce-8a97-2f69a224d73f",
+   "metadata": {},
+   "source": [
+    "## Quantize the model"
```

Comment:
where are the 2 functions you mention: "Now, let's create two functions..."
maybe this title "## Quantize the model" is not in place, should be later on